src/datasources/cache/cache.router.ts

```diff
@@ -153,10 +153,7 @@ export class CacheRouter {
     chainId: string;
     moduleAddress: string;
   }): CacheDir {
-    return new CacheDir(
-      this.getSafesByModuleCacheKey(args), // TODO: Refactor other `CacheDir` keys to use respective key methods
-      '',
-    );
```

Comment:
I think it's due to this PR: https://github.com/safe-global/safe-client-gateway/pull/960 But yeah, being strict it can be removed in that PR instead 😄

Reply:
Due to the order of PRs, #960 was merged before this so the note was no longer needed.